packages/components/README.md

```diff
@@ -9,7 +9,8 @@ A package containing the components for the HashiCorp Design System.
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.24 or above
+* Ember.js v3.28 or above
+  * Note: The library _should_ work with earlier versions of Ember, but we only test with Ember 3.28 and newer
 * Ember CLI v3.24 or above
```

Comment:
Should this version align as well?

Reply:
That's a good question. In practice my experience is that ember versions are almost entirely decoupled from ember-cli and we don't have any way of testing what the oldest version of ember-cli that would work with our components is either. It almost makes me feel like taking out the ember-cli line makes the most sense. What do you think?

Reply:
Since we have custom blueprints, we should probably leave it in, even though that relates more to creating components than using them.

Reply:
I updated the ember-cli lines to also state 3.28